<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PSF_XYZ_Dialog_BF</class>
 <widget class="QDialog" name="PSF_XYZ_Dialog_BF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PSF-XYZ Automatic Detection BF</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>712</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="12" column="0">
     <widget class="QCheckBox" name="checkBox_SavePeaks">
      <property name="text">
       <string>Save PSF Peaks FWHM</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_channel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select Channel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_objNA">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>NA</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="SpinBox_channel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="ExEm_text">
      <property name="text">
       <string>n.a.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="6">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QDoubleSpinBox" name="SpinBox_NA">
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.700000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QLineEdit" name="text_filename">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>imagefile</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="OpenFile">
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_objname">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Objective</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_threshold">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Threshold used to identify the peaks.</string>
      </property>
      <property name="text">
       <string>Threshold [0-1]</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="SpinBox_threshold">
      <property name="toolTip">
       <string/>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_pixsize">
      <property name="toolTip">
       <string>Pixel Size in image plane.</string>
      </property>
      <property name="text">
       <string>Pixel Size [nm]</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="SpinBox_pixsize">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="zspacing">
      <property name="toolTip">
       <string>Distance between 2 focal planes.</string>
      </property>
      <property name="text">
       <string>Z-Spacing [micron]</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_guess_fwhmz">
      <property name="toolTip">
       <string>Guess for FWHM-Z used tfor the Gauss Fit.</string>
      </property>
      <property name="text">
       <string>Guess FWHM-Z [micron]</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDoubleSpinBox" name="SpinBox_guess_fwhmz">
      <property name="toolTip">
       <string/>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>5.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QDoubleSpinBox" name="SpinBox_zspacing">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.002500000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QCheckBox" name="writexls">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Write Data to Excel File</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_ExEm">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>EX/EM Wavelength [nm]</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="objname_text">
      <property name="text">
       <string>n.a.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_subimage_size">
      <property name="toolTip">
       <string>Size of subimage which is cut out to apply the 2D Gauss Fit.</string>
      </property>
      <property name="text">
       <string>Subimage Size [pixel]</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QSpinBox" name="SpinBox_subimage_size">
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="label_offset">
      <property name="toolTip">
       <string>Size of subimage which is cut out to apply the 2D Gauss Fit.</string>
      </property>
      <property name="text">
       <string>Subtract Offset CCD [cts]</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QSpinBox" name="SpinBox_offset">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>50000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QCheckBox" name="checkBox_SaveOrthoView">
      <property name="text">
       <string>Save PSF OrthoView</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QCheckBox" name="checkBox_SavePSFVolume">
      <property name="text">
       <string>Save PSF Volume</string>
      </property>
     </widget>
    </item>
    <item row="12" column="4" colspan="2">
     <widget class="QPushButton" name="pushButton_StartCalc">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start Detection</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="6">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_Immersion">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Immersion</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="immersion_text">
      <property name="text">
       <string>n.a.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_RI">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ref. Index</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="ri_text">
      <property name="text">
       <string>n.a.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
